supernode/daemon/mgr/gc/gc_task.go

```diff
@@ -49,17 +49,15 @@ func (gcm *Manager) gcTasks(ctx context.Context) {
 			continue
 		}
 
-		if !gcm.gcTask(ctx, taskID) {
-			continue
-		}
+		gcm.gcTask(ctx, taskID)
 		removedTaskCount++
 	}
 
 	logrus.Infof("gc tasks: success to full gc task count(%d), remainder count(%d)", removedTaskCount, totalTaskNums-removedTaskCount)
 }
 
-func (gcm *Manager) gcTask(ctx context.Context, taskID string) bool {
-	logrus.Infof("start to gc task: %s", taskID)
+func (gcm *Manager) gcTask(ctx context.Context, taskID string) {
+	logrus.Infof("start gc task: %s", taskID)
```

Comment:
IMOP,  the correct gramma should be `start to do something`.

Reply:
`
logrus.Infof("gc tasks: success to full gc task count(%d), remainder count(%d)", removedTaskCount, totalTaskNums-removedTaskCount)
`
I think add a info-prefix may be better like this,and logrus.Debug() may be better